public/app/features/alerting/unified/types/receiver-form.ts

```diff
@@ -26,7 +26,7 @@ export interface CloudChannelValues extends ChannelValues {
 export interface GrafanaChannelValues extends ChannelValues {
   type: NotifierType;
   provenance?: string;
-  disableResolveMessage: boolean;
+  disableResolveMessage?: boolean;
```

Comment:
This isn't always returned from the k8s API, it's actually optional, but we're using the same type